src/lib/wasm-webworker.ts

```
@@ -0,0 +1,156 @@
+		const wasm_js = await import(
+			/* webpackIgnore: true */
+			'/wasm/wasm.js' as string
+		);
+		const wasmModule = (await wasm_js.default()) as WasmExports;
+		module = {
+			module: wasmModule,
+			cwrapped: wasmModule.cwrap('luau_template', 'number', ['string', 'string'])
+		};
+		return module;
+	} catch (error) {
+		throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)